tests/test_pygfx_renderer.py

```diff
@@ -49,20 +49,28 @@ def test_pygfx_insert_remove():
     child1 = gfx.WorldObject()
     child2 = gfx.WorldObject()
 
+    def equals(a, b):
+        if len(a) != len(b):
+            return False
+        for x, y in zip(a, b):
+            if x != y:
+                return False
+        return True
+
     renderer.insert(child1, parent)
-    assert parent.children == (child1,)
```

Comment:
Why doesn't this work anymore?

Reply:
Because children is now a list instead of a tuple

Reply:
I decided to add the `equals` function here, such that if pygfx would ever change the type again, it wouldn't break this test.